install/installer/pkg/components/slowserver/deployment.go

```diff
@@ -385,7 +393,26 @@ func deployment(ctx *common.RenderContext) ([]runtime.Object, error) {
 							},
 							volumes...,
 						),
-						InitContainers: []corev1.Container{*common.DatabaseWaiterContainer(ctx), *common.MessageBusWaiterContainer(ctx)},
```

Comment:
This illustrates some of the less-pleasant aspects of using global variables to derive the init container. Alternative would be to parametrize the function with the DB context but that's a bigger change

Reply:
Same comment as above about generalisation not being worth it here.